tests/manager/core/test_config.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from sqlalchemy.exc import ArgumentError
+
+from palace.manager.core.config import CannotLoadConfiguration, Configuration
+from palace.manager.service.logging.configuration import LogLevel
+
+
+class TestConfiguration:
+    def test_database_url(
```

Comment:
Since `database_url` is mocked out now for most of our tests, add a small test for it, so we don't lose coverage of it.